pom.xml

```diff
@@ -38,8 +38,8 @@
     <java.version>1.8</java.version>
     <maven.version>3.6.3</maven.version>
     <spotbugs-annotations.version>4.0.2</spotbugs-annotations.version>
-    <slf4j.version>1.7.25</slf4j.version>
-    <logback.version>1.2.3</logback.version>
+    <slf4j.version>1.7.32</slf4j.version>
+    <logback.version>1.2.11</logback.version>
```

Comment:
I think I'd prefer this if the version range was here... and then the dependency declarations just referenced this value.

Reply:
The reason I like that is it avoids repeating the upper bounds everywhere. Maybe two variables:
```xml
<slf4j.base.version>1.7.32</slf4j.base.version>
<slf4j.range.version>[${slf4j.version},1.7.999)</slf4j.range.version>
```

Reply:
I can go with that.  Will change.